Tutorials/Processing/11_video/sketch_11_10_BlobTracking_lifespan/sketch_11_10_BlobTracking_lifespan.pde

```diff
@@ -145,7 +145,6 @@ void draw() {
       }
       if (matched != null) {
         matched.taken = true;
-        matched.lifespan = maxLife;
```

Comment:
I'm good with this fix, however, it might be nice to comment it out with a note for people who are following along with the video?